docs/context/compaction.md

```diff
@@ -0,0 +1,63 @@
+# Compress workflow context for performance
+
+As an ADK agent runs a workflow it builds up information, including user
```

Comment:
i don't know if it's specific to "workflow". see if there is a more generic word.